scripts/prod/env_writer_prod.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Make sure the script can only be run via SocialPredict Script
+[ -z "$CALLED_FROM_SOCIALPREDICT" ] && { echo "Not called from SocialPredict"; exit 42; }
+
+# Function to check if the password is strong
+check_password_strength() {
+    local password=$1
+    if [[ ${#password} -ge 8 && "$password" =~ [A-Za-z] && "$password" =~ [0-9] && "$password" =~ [^A-Za-z0-9] ]]; then
+        return 0  # Strong password
+    else
+        echo "Password must be at least 8 characters long, include letters, numbers, and special characters."
+        return 1  # Not strong enough
+    fi
+}
+
+check_username_strength() {
+    local username=$1
+    if [[ ${#username} -ge 5 && "$username" =~ [^A-Za-z0-9] ]]; then
+        return 0  # Strong username
+    else
+        echo "Username must be at least 5 characters long and include special characters."
+        return 1  # Not strong enough
+    fi
+}
+
+# Function to create and update .env file
+init_env() {
+	# Create .env file
+	cp "$SCRIPT_DIR/.env.example" "$SCRIPT_DIR/.env"
+
+	# Update .env file
+
+	# Update APP_ENV
+	sed -i -e "s/APP_ENV=.*/APP_ENV=production/g" .env
+
+	# Update domain name
+	read -r -p "What domain do you wish to use for the application? " domain_answer
+	while [ -z "$domain_answer" ]
+	do
+		echo "You need to specify a domain."
+		read -r -p "What domain do you wish to use for the application? " domain_answer
+	done
+
+	# Change the Domain setting:
+	sed -i -e "s/DOMAIN=.*/DOMAIN='$domain_answer'/g" .env
+	echo "Setting DOMAIN to: $domain_answer"
+
+	echo
+
+	# Update email address
+	read -r -p "What email address do you wish to use for the SSL Certificate? " email_answer
+	while [ -z "$email_answer" ]
+	do
+		echo "You need to specify an email address."
+		read -r -p "What email address do you wish to use for the SSL Certificate? " email_answer
+	done
+
+	# Change the Email setting:
+	sed -i -e "s/EMAIL=.*/EMAIL='$email_answer'/g" .env
+	echo "Setting EMAIL to: $email_answer"
+
+	echo
+
+	# Update Database User:
+	while true; do
+		read -r -p "Specify a username for the Database User (default is insecure and not allowed): " db_user_answer
```

Comment:
Requiring special characters for postgres username, which seems not right.

Seems like we should only require letters and numbers, either upper and lowercase.

```
Specify a username for the Database User (default is insecure and not allowed): j711fMI1KEXh9GJA
Username must be at least 5 characters long and include special characters.
Specify a username for the Database User (default is insecure and not allowed): j711fMI1KEXh&
Setting Database User to: j711fMI1KEXh&

```

Reply:
commit [d144570](https://github.com/openpredictionmarkets/socialpredict/commit/d1445701df1cda5d37856e9d4c8620aad6eb0367) fixes that to only require 5 chars long characters or numbers.